docs/docs/10-reference.md

```diff
@@ -216,6 +218,26 @@ Options:
 - **`buildOptions.webModulesUrl`** | `string` | Default: `web_modules`
   - Rename your web modules directory.
 
+#### `config.testOptions`
+
+`object` (options)
+
+Settings that determine how the Snowpack test environment behaves.
+
+Example:
+
+```js
+testOptions: {
+  files: ['my-test-dir/*.test.js'];
+}
+```
+
+Options:
+
+- **`testOptions.files`** | `string[]` | Default: `["__tests__/**/*", "**/*.@(spec|test).*"]`
```

Comment:
Just my opinion: I’d put a link here stating that this pattern uses [glob’s `ignore` option](https://www.npmjs.com/package/glob#options). For me, I’m always confused what I can/can’t do when it comes to glob patterns like this.

Reply:
ah agreed, but not sure if it makes sense to document the ins and out of globs each time we use them. Maybe have some overall glob support documentation?